### ISSUE TITLE
Could not find module 'nativescript-firebase/firebase-common'

### ISSUE THREAD

**Issue by jsrbn** [EddyVerbruggen/nativescript-plugin-firebase#689]
When importing the plugin into a component with `import firebase from 'nativescript-plugin-firebase`, I retrieve the following issue: 

`file:///app/app.js:1:228301: JS ERROR Error: Could not find module 'nativescript-plugin-firebase/firebase-common'. Computed path '/Users/eykrehbein/Library/Developer/CoreSimulator/Devices/D5673E61-F2C2-4B8F-AD84-27AC6D1AE8B5/data/Containers/Bundle/Application/5E9E849A-1DFE-4466-B574-DEFEBB22B3C5/dist.app/app/tns_modules/nativescript-plugin-firebase/firebase-common'.`.

In a nutshell, the plugin couldn't find the `firebase-common` module even it's right there in the main directory of 'nativescript-plugin-firebase`

**Comment by EddyVerbruggen** [comment]
Feel free to reopen, but please share a repo demonstrating the issue if you do.
